amadeus-parquet/Cargo.toml

```diff
@@ -37,7 +37,7 @@ rustversion = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_closure = "0.3"
 snap = "1.0"
-sum = "0.1"
+sum = { version = "0.1.7", default-features = false, features = ["futures", "serde", "2", "3"]  }
```

Comment:
The "futures" and "serde" features weren't being used before so we don't need to add them